test/fixtures.rb

```diff
@@ -40,6 +40,10 @@ class Application < Hanami::Application
 
       default_response_format :html
 
+      security.x_frame_options "DENY"
+      security.content_security_policy "form-action 'self'; referrer origin-when-cross-origin; reflected-xss block; manifest-src 'self'; frame-ancestors 'self'; base-uri 'self'; default-src 'none'; connect-src 'self'; img-src 'self'; style-src 'self'; font-src 'self'; object-src 'self'; plugin-types application/pdf application/x-shockwave-flash application/x-java-applet; child-src 'self'; media-src 'self'"
```

Comment:
> Do we have this legacy problem with Hanami?
No, we don't. 

> I think we can safely remove CSPRO support.
Sure, we can do that.

> In case someone needs it, they can always use: 1. Default headers 2. a shared module for each action (hint controller.prepare) and always send that report-only header.
:+1: